lib/active_fedora/file.rb

```diff
@@ -134,6 +134,10 @@ def metadata
       @metadata ||= ActiveFedora::WithMetadata::MetadataNode.new(self)
     end
 
+    def checksum
+      ActiveFedora::Checksum.new(self)
```

Comment:
Personally I don't think the operation cost warrants the complexity of caching.  Or if it does, perhaps it belongs on `ActiveFedora::File::Attributes#digest`.  @jcoyne